docs/modules/ROOT/pages/reference.adoc

```diff
@@ -146,3 +146,20 @@ Appears In: xref:{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-c
 
 
 
+[id="{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-controller-api-v1alpha1-source"]
+==== Source
+
+Source defines policies and data that are evaluated together
+
+[quote]
+Appears In: xref:{anchor_prefix}-github-com-hacbs-contract-enterprise-contract-controller-api-v1alpha1-enterprisecontractpolicyspec[$$EnterpriseContractPolicySpec$$]
+
+[cols="25a,75a", options="header"]
+|===
+| Field | Description
+| *`name`* __string__ | Name of the source
+| *`policy`* __string array__ | One or more policy rules
+| *`data`* __string array__ | List of data sources for the policy rules
```

Comment:
Maybe:
```
.... One or more go-getter style policy source urls
.... Zero or more go-getter style policy data source urls
```

Reply:
Or...
```
.... List of policy rule sources
.... List of data sources for the policy rules
```

Reply:
Combined the two. Let me know what you think!